service/src/main/kotlin/fi/espoo/evaka/messaging/daycarydailynote/DaycareDailyNoteQueries.kt

```diff
@@ -109,3 +109,25 @@ fun Database.Transaction.deleteChildDaycareDailyNotes(childId: UUID) {
         .bind("id", childId)
         .execute()
 }
+
+fun Database.Transaction.deleteExpiredDaycareDailyNotes() {
+    createUpdate(
+        """
+WITH expired_child_notes AS (
+    SELECT id, child_id
+    FROM daycare_daily_note
+    WHERE child_id IS NOT null
+        AND modified_at < now() - INTERVAL '12 hours'
```

Comment:
Did we decide that we want to replace these `now()` calls with overridable arguments (for testing)?